client/my-sites/site-settings/form-discussion.jsx

```diff
@@ -179,6 +179,12 @@ class SiteSettingsFormDiscussion extends Component {
 					onChange={ this.handleCommentOrder }>
 					<span>{ translate( 'Comments should be displayed with the older comments at the top of each page' ) }</span>
 				</CompactFormToggle>
+				<CompactFormToggle
+					checked={ fields[ 'gravatar-hovercards' ] }
```

Comment:
This is what happens when I review code before I finish my morning coffee. 😆 

But yup, `JetpackModuleToggle` will solve it. :+1: